packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotFooter/ChatbotFooter.md

```diff
@@ -39,13 +40,15 @@ Footnotes can be static text or a button which triggers a popover.
 ```
 
 ### Message bar with speech to text
-By default the message bar enables uploading files. Setting the `hasAttachButton` to `false` will disable that feature.
+
+By default the message bar enables uploading files. Setting the `hasAttachButton` to `false` will disable that feature. The speech to text button will only appear if `'SpeechRecognition'` or `'webkitSpeechRecognition'` are available in `window`.
```

Comment:
The button doesn't show on Firefox, so adding this to make it more clear hopefully.

Reply:
Is it recommended that the consumer check to make sure either of those exist in window before rendering the button? should we demonstrate that in the example?

Reply:
That's the existing logic in the button - it doesn't show if those don't exist (which they don't, in Firefox). If you look at the example in Firefox, it talks about speech to text, but there's only an attach button. It shows in Chrome and Safari though. Do you want to explicitly call out that it doesn't work in Firefox right now?